docs/src/tutorial/simple_system.md

```diff
@@ -24,19 +26,54 @@ system on Spine Toolbox.
 
 ### Installation and upgrades
 
-If you haven't yet installed the tools yet, please follow the installation guides: 
-- For Spine Toolbox: [Spine Toolbox installation guide](https://github.com/spine-tools/SpineOpt.jl#installation)
-- For SpineOpt: [SpineOpt installation guide](https://github.com/spine-tools/Spine-Toolbox#installation)
+If you haven't yet installed the tools or you are not sure whether you have the latest version,
+please follow the installation/upgrade guides: 
+- For Spine Toolbox: [Spine Toolbox installation guide](https://github.com/spine-tools/Spine-Toolbox#installation)
+- For SpineOpt: [SpineOpt installation guide](https://github.com/spine-tools/SpineOpt.jl#installation)
 
-If you are not sure whether you have the latest version, please upgrade to ensure compatibility with this guide.
+## Guide
 
-- For Spine Toolbox:
-    - If installed with pipx, then use `python -m pipx upgrade spinetoolbox`
-    - If installed from sources using git, then `git pull`, `python -m pip install -U -r requirements.txt`
+### Spine Toolbox workflow
 
-- For SpineOpt: [SpineOpt upgrade guide](https://github.com/spine-tools/SpineOpt.jl#upgrading)
+The workflow for this tutorial is quite simple: A SpineOpt tool that 
+reads data from an input database, executes the simulation/optimization 
+and writes the results to an output database.
 
-## Guide
+It is almost as simple as dragging these items (i.e. Data Store and Run SpineOpt) 
```

Comment:
This is a very chill way to say this, but the first time I read it I didn't realize I actually needed to do these steps until later. I think it could be a bit more clear.

Reply:
I've adjusted this very slightly but it may be enough to make it clearer that they should create the workflow.